app/Main.hs

```diff
@@ -412,23 +412,44 @@ checkForUpdates applyMinorUpdates applyMajorUpdates = do
     isMinorReleaseFrom (x : xs) (y : ys) = y == x && ys > xs
     isMinorReleaseFrom _        _        = False
 
+verify :: Maybe String -> IO ()
+verify Nothing = echoT "A package name is required for verify"
+verify (Just inputName) = case mkPackageName (pack inputName) of
+  Left pnError -> echoT . pack $ "Error while parsing input package name: " <> show pnError
+  Right name -> do
+    pkg <- readPackageFile
+    db <- readPackageSet pkg
+    case name `Map.lookup` db of
+      Nothing -> echoT . pack $ "No packages found with the name " <> show (runPackageName $ name)
+      Just _ -> do
+        dependencies <- map fst <$> getTransitiveDeps db [name]
```

Comment:
This should be reverse dependencies.

Reply:
What do you mean? Right now if I feed "aff", it checks "aff" first and then goes down the list alphabetically to "unsafe-coerce". Should I reverse this list?

Reply:
No, I mean we should test things which depend on `aff`, not things which `aff` depends on.

Reply:
oh, oops...